src/main/resources/jenkins/branch/MultiBranchProjectWelcomeView/main.jelly

```diff
@@ -22,10 +22,21 @@
  ~ THE SOFTWARE.
  -->
 <?jelly escape-by-default='true'?>
-<st:compress xmlns:st="jelly:stapler">
-  <p>
-    This project scans branches in your SCM and generate a job for each of them,
-    but you have no branches configured.
-    <a href="./configure">Configure the project</a> and then <a href="./indexing">index branches</a> to get started.
-  </p>
+<st:compress xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
+  <div style="margin: 20px 0; padding: 10px 30px 20px 30px; background: #f8f8f8; border-radius: 10px; max-width: 55em;">
+    <h1>This folder is empty</h1>
+
+    <p>
+        There are no branches found that contain buildable projects.
+        Jenkins automatically builds and manages projects in branches that
+        contain recognizable projects.
+    </p>
+    
+    <st:include it="${it.owner.projectFactory}" page="getting-started" optional="true"/>
```

Comment:
:ant: `MultiBranchProjectFactory` class Javadoc should mention that it may have a view named `getting-started`.